src/Language/PureScript/CoreFn/ToJSON.hs

```diff
@@ -64,8 +64,8 @@ moduleToJSON v m = object [ T.pack "imports"   .= map (annToJSON *** moduleNameT
                           ]
 
 bindToJSON :: Bind Ann -> Value
-bindToJSON (NonRec ann n e) = object [ runIdent n .= ( annToJSON ann, exprToJSON e ) ]
-bindToJSON (Rec bs) = object $ map (\((ann, n), e) -> runIdent n .= ( annToJSON ann, exprToJSON e ) ) bs
+bindToJSON (NonRec ann n e) = toJSON [(runIdent n, annToJSON ann, exprToJSON e)]
+bindToJSON (Rec bs) = toJSON $ map (\((ann, n), e) -> (runIdent n, annToJSON ann, exprToJSON e)) bs
```

Comment:
I don't understand why this needs to change. It's a recursive group, so the order should be unimportant.